src/cli.ts

```diff
@@ -17,7 +17,7 @@ main({
 })
   .then((isSuccessful) => {
     if (!isSuccessful) {
-      process.exitCode = 1;
+      process.exitCode = 100;
```

Comment:
I've changed this exit code so that we can distinguish between the case in which `module-lint` fails because one of the rules failed or because an unknown error occurred.

If we did not have this change here, the workflow would halt if any project failed lint. We don't want that to happen; we want the workflow to continue. We account for this special exit code in `.github/script/run-module-lint.sh` and exit with 0 if `module-lint` exits with 100.